elements/pf-background-image/pf-background-image.ts

```diff
@@ -0,0 +1,122 @@
+ * @summary Allows users to place an image in the background of your page or area of a page.
+ *
+ * @slot filter - Overrides the default svg filter for the background image.
+ * @slot - Default content slot.
+ *
+ * @cssprop {<color>} --pf-c-background-image--BackgroundColor {@default `#151515`}
+ * @cssprop --pf-c-background-image--BackgroundImage
+ * @cssprop --pf-c-background-image--BackgroundImage-2x
+ * @cssprop --pf-c-background-image--BackgroundImage--sm
+ * @cssprop --pf-c-background-image--BackgroundImage--sm-2x
+ * @cssprop --pf-c-background-image--BackgroundImage--lg
+ * @cssprop --pf-c-background-image--Filter {@default `url("#filter")`}
+ *
+ */
+@customElement('pf-background-image')
+export class PfBackgroundImage extends LitElement {
+  static readonly styles = [styles];
+
+  /**
+   * The URL for the image shown on mobile
+   */
+  @property({ reflect: true }) src?: string;
+
+  /**
+   * The image shown on mobile with 2x DPI
+   */
+  @property({ reflect: true, attribute: 'src-2x' }) src2x?: string;
+
+  /**
+   * The URL for the image shown on small screens (min-width: 576px)
+   */
+  @property({ reflect: true, attribute: 'src-sm' }) srcSm?: string;
+
+  /**
+   * The URL for the image shown on small screens (min-width: 576px) with 2x DPI
+   */
+  @property({ reflect: true, attribute: 'src-sm-2x' }) srcSm2x?: string;
+
+  /**
+   * The URL for the image shown on large screens (min-width: 992px)
+   */
+  @property({ reflect: true, attribute: 'src-lg' }) srcLg?: string;
+
+  /**
+   * Apply SVG Filter to the image
+   */
+  @property({ type: Boolean, reflect: true }) filter = false;
+
+  @queryAssignedElements({ slot: 'filter', selector: 'svg' }) private _svg?: SVGElement[];
+
+  #svg?: SVGElement;
+
+  #updated = false;
+
+  render() {
+    return html`
+      <div id="container" part="container" style="${this.#cssVars()}">
+        ${!this.filter ? html`` : html`
+          <slot name="filter" @slotchange=${this.#onSlotChange}>
+
+              ${(this.#svg && this.#updated) ? this.#svg : html`
+                <svg xmlns="http://www.w3.org/2000/svg" width="0" height="0">
+                  <filter id="filter">
+                    <feColorMatrix type="matrix" values="1 0 0 0 0 1 0 0 0 0 1 0 0 0 0 0 0 0 1 0"></feColorMatrix>
+                    <feComponentTransfer color-interpolation-filters="sRGB" result="duotone">
+                      <feFuncR type="table" tableValues="0.086274509803922 0.43921568627451"></feFuncR>
+                      <feFuncG type="table" tableValues="0.086274509803922 0.43921568627451"></feFuncG>
+                      <feFuncB type="table" tableValues="0.086274509803922 0.43921568627451"></feFuncB>
+                      <feFuncA type="table" tableValues="0 1"></feFuncA>
+                    </feComponentTransfer>
+                  </filter>
+                </svg>
+              `}
+          </slot>
+        `}
+        <div class="content" part="content">
```

Comment:
do we need the wrapper div? why not `<slot id="content" part="content">` and set `display: block`?

Reply:
Mostly likely could do that, trying to remember what I ran into to add the wrap in the first place.  Might have been the question of needing the content slot in the first place.  I don't think PF implementation shows that idea in code IIRC / part of the first todo of determining if it even needs to exist.  But yes, wouldn't see why this suggested change wouldn't work.